Makefile

```diff
@@ -4,6 +4,8 @@ CGO_ENABLED=0
 GOOS=linux
 CORE_IMAGES=./cmd/activator ./cmd/autoscaler ./cmd/autoscaler-hpa ./cmd/controller ./cmd/queue ./cmd/webhook ./vendor/knative.dev/pkg/apiextensions/storageversion/cmd/migrate ./cmd/domain-mapping ./cmd/domain-mapping-webhook
 TEST_IMAGES=$(shell find ./test/test_images ./test/test_images/multicontainer -mindepth 1 -maxdepth 1 -type d)
+# Exclude wrapper images like multicontainer and initcontainers as those are just ko convenience wrappers used upstream. The openshift serverless tests use other images to run.
+TEST_IMAGES_WITHOUT_WRAPPERS=$(shell find ./test/test_images ./test/test_images/multicontainer -mindepth 1 -maxdepth 1 -type d -not -name multicontainer -not -name initcontainers)
```

Comment:
I suppose this variable should be used also on line 24 in [this line](https://github.com/openshift-knative/serving/pull/109/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R24)
If I run `make test-install` on my localhost I can see this:
```
make test-install
for img in ./test/test_images/autoscale ./test/test_images/failing ./test/test_images/grpc-ping ./test/test_images/hellohttp2 ./test/test_images/hellovolume ./test/test_images/helloworld ./test/test_images/httpproxy ./test/test_images/initcontainers ./test/test_images/multicontainer ./test/test_images/pizzaplanetv1 ./test/test_images/pizzaplanetv2 ./test/test_images/readiness ./test/test_images/runtime ./test/test_images/singlethreaded ./test/test_images/timeout ./test/test_images/volumes ./test/test_images/wsserver ./test/test_images/multicontainer/servingcontainer ./test/test_images/multicontainer/sidecarcontainer; do \
	go install $img ; \
done
no Go files in /home/mgencur/go/src/knative.dev/serving/test/test_images/initcontainers
no Go files in /home/mgencur/go/src/knative.dev/serving/test/test_images/multicontainer
```

Otherwise this PR looks ok.